web/client/plugins/manager/ManagerMenu.jsx

```diff
@@ -0,0 +1,105 @@
+/**
```

Comment:
Not too much, only the visibility (admin only) and the possibility to configure it manually.
In the future we could consider to make 2 instances of the burger menu with different tools wired inside it.